composer.json

```diff
@@ -16,7 +22,8 @@
         "drush/drush": "^11|^12|^13",
         "symfony/yaml": "^6|^7",
         "twig/twig": "^3",
-        "vlucas/phpdotenv": "^4|^5"
+        "vlucas/phpdotenv": "^4|^5",
+        "lullabot/composer-checks": "@dev"
```

Comment:
Let's create a tag we can point to instead.

https://github.com/Lullabot/composer-checks/issues/4